cfgov/unprocessed/apps/tccp/js/index.js

```diff
@@ -26,6 +26,8 @@ function init() {
   orderingDropdown.move();
   // Initialize any tooltips on the page
   initializeTooltips();
+  //Reinitialize tooltips after an htmx request replaces DOM nodes
+  document.addEventListener('htmx:afterRequest', initializeTooltips);
```

Comment:
To match the other mumbo jumbo in this file, how do you feel about:

```suggestion
  // Reinitialize tooltips after an htmx request replaces DOM nodes
  attach(document, 'htmx:afterRequest', initializeTooltips);
```